### DISCUSSION TITLE
Fullscreen image? Available in QCarousel but not QImg. Strategy for creating?

### DISCUSSION THREAD

**Discussion by KingWebsites** [quasarframework/quasar#14552] [General - Components / Directives / etc] [answered]
Hi, I'm looking to add a fullscreen button to images on the website. I am currently using QImg.

I noticed that QCarousel does have this functionality https://quasar.dev/vue-components/carousel#fullscreen but QImg does not.

As a strategy for solving this issue is it better to enhance the QImg component using code from the QCarousel? Or is the prefered Quasar method to create a brand new QImgFullscreen plugin? I'd be grateful if somebody could let me know roughly how they would approach this problem.

Thank you.

**Comment by MilosPaunovic** [comment, marked as answer]
https://codepen.io/milospaunovic/pen/xxjJgrB?editors=1010

**Reply by KingWebsites** [reply]
Thank you. Of course, it's the Quasar class of "fullscreen" that can do all the heavy lifting and can be applied to any object I like.

Thanks for your input, that's really saved me some time today.
